src/Client.php

```diff
@@ -259,6 +259,17 @@ protected function getForceIpv4Flag(ConnectionInterface $connection)
     }
 
     /**
+     * Extracts the value of the allow-self-signed option from a given connection.
+     *
+     * @param \Phergie\Irc\ConnectionInterface $connection
+     * @return boolean TRUE to allow self signed certificates, FALSE otherwise
+     */
+    protected function getAllowSelfSignedFlag(ConnectionInterface $connection)
+    {
+        return (boolean) $connection->getOption('allow-self-signed') ?: false;
```

Comment:
Sorry, just noticed that Phergie configuration uses camel case (e.g. `serverHostname`) rather than this style (i.e. `allow-self-signed`). Would you mind updating this to use camel case as well?